avocado/utils/astring.py

```diff
@@ -219,7 +219,7 @@ def string_safe_encode(string):
     try:
         return string.encode("utf-8")
     except UnicodeDecodeError:
-        return string.decode("utf-8").encode("utf-8")
+        return string.decode("utf-8", "ignore").encode("utf-8")
```

Comment:
@ldoktor the issue that motivated this has been fixed directly on avocado-vt.  I'm fine with this, but I'd like to check if you still want this merged.

Thanks!